README.md

```diff
@@ -72,7 +76,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+[:friends].each do
+  |friend| puts "Belle is friends with #{name}"
```

Comment:
Common practice is to move the iteration variable (|friend|) to the first line of the each statement. So it should look like...

```rb
friends.each do |friend|
  puts "Belle is friends with #{name}"
end
```